tools/permuter_settings.toml

```diff
@@ -1,3 +1,5 @@
+compiler_type = "ido"
```

Comment:
Was this added unintentionally?

Reply:
This is a new permuter thing, it uses the compiler to weight and decide what randomisations to apply. Someone should probably PR it separately, I've just been adding it to all my PRs so it gets in when one of them does.